compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -680,7 +693,6 @@ object Scanners {
           currentRegion = Indented(nextWidth, COLONeol, currentRegion)
           offset = next.offset
           token = INDENT
-    end observeIndented
```

Comment:
+1, I also don't like these end markers 😄

Reply:
the method used to be longer and had a blank line. I must have deleted it during other edits while experimenting, but here it's no longer needed.